docs/articles/deploying-from-private-registries.md

```diff
@@ -1,26 +1,6 @@
-# Using Testkube with Private Registries  
+# Using Global Parameters in Testkube Helm Charts  
 
-This guide shows how to deploy and use Testkube with images from private registries. 
```

Comment:
why are we removing this? don't users often ask about this?

Reply:
It's only one line in `values.yaml` file now:
```
global:
   imageRegistry: registry
```

Before we had workarounds for third-party charts that users had to implement.